CondTools/RunInfo/plugins/LHCInfoPerFillPopConAnalyzer.cc

```diff
@@ -255,8 +249,13 @@ class LHCInfoPerFillPopConSourceHandler : public popcon::PopConSourceHandler<LHC
     cond::Time_t lastSince = tagInfo().lastInterval.since;
     if (tagInfo().isEmpty()) {
       // for a new or empty tag, an empty payload should be added on top with since=1
-      addEmptyPayload(1);
-      lastSince = 1;
+      if (m_endFillMode) {
+        addEmptyPayload(1);
+        lastSince = 1;
+      } else {
+        addEmptyPayload(cond::time::lumiTime(1, 1));
+        lastSince = cond::time::lumiTime(1, 1);
```

Comment:
If we assign a lumiid type value to lastSince we cannot assign it's value to targetSince anymore in https://github.com/cms-sw/cmssw/pull/43837/files#diff-cf7bbd18a999d3e00d1efbf718cb2170276b4688a7e28bdb058b7cb845484a1cR271 as it is required to be of type timestamp (it is later converted to boost time).
I've fixed it on this branch, you can cherry-pick the top comment: https://github.com/CTPPS/cmssw/tree/jc_duringFill_to_lumiid_targetTime_fix

Reply:
Done in [96cce8c](https://github.com/cms-sw/cmssw/pull/43837/commits/96cce8c6db09cabb4a9cf6e6d19b3e6e20b9b599), I cherry-picked your commit but then I had to fix the code-format, so that's why it's appearing as a different commit wrt yours.